### DISCUSSION TITLE
stream camera output

### DISCUSSION THREAD

**Discussion by sehadnassim** [cyberbotics/webots#5778] [General]
Is it possible to stream the robots camera through ffmpeg or gstreamer ?

**Comment by BenjaminDeleze** [comment]
Not directly but you can see the camera through robot windows like in this example: https://webots.cloud/run?version=R2022b&url=https://github.com/cyberbotics/webots-cloud-simulation-examples/blob/main/2_compile_controller/worlds/spot.wbt&type=demo

To see the camera:

- Play the simulation.
- Open the robot window by clicking on the robot window button in the toolbar or press `w`.
- Go to the camera tab.

If you need only the camera, you can build a personalized robot window with just the camera inside
